app/uploaders/image_uploader.rb

```diff
@@ -35,6 +35,10 @@ def store_dir
   #   process :resize_to_fit => [50, 50]
   # end
 
+  # version :thumb do
+  #   process :resize_to_limit => [50, 50]
+  # end
+
```

Comment:
This was boilerplate stuff from the image uploader that we tried to use, but required another plug-in to work that seems to be a major pain to install properly (rmagick) so we backed it out again.